shared/feature/profile/src/main/java/com/epmedu/animeal/profile/presentation/ui/ProfileInputForm.kt

```diff
@@ -22,15 +33,20 @@ import com.epmedu.animeal.profile.presentation.ProfileInputFormEvent.SurnameChan
 import com.epmedu.animeal.profile.presentation.viewmodel.ProfileState
 import com.epmedu.animeal.profile.presentation.viewmodel.ProfileState.FormState.READ_ONLY
 import com.epmedu.animeal.resources.R
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
 @Composable
 fun ProfileInputForm(
+    modifier: Modifier = Modifier,
     state: ProfileState,
-    onEvent: (ProfileInputFormEvent) -> Unit,
-    modifier: Modifier = Modifier
+    bottomSheetState: ModalBottomSheetState? = null,
```

Comment:
I don't think this composable needs to know about the `bottomSheetState`. Can we just pass a callback instead?